services/persistence/main.py

```diff
@@ -1,31 +1,94 @@
 # -*- coding: utf-8 -*-
+import json
 import os
-import manifesto_data
-
-from flask import Flask, jsonify, request
 import sqlite3
 import urllib.parse
 
+from flask import Flask, jsonify, request
+from manifesto_data import ManifestoDataLoader
+
+
 app = Flask(__name__)
 
 DEBUG = True  # os.environ.get('DEBUG') is not None
 VERSION = 0.1
 
+DB_FILENAME = '/db/test.db'
+print('Using database', DB_FILENAME)
+
 
 @app.route("/", methods=['POST'])
 def index():
     return jsonify({})
 
 
+@app.route("/texts_and_labels", methods=['POST'])
+def texts_and_labels():
+    """
+    stores the POST-body texts and labels.
+
+    expe                                                                                                                                                                                                            cts a POST-body in the format:
+    {
+        "data": [
+            {"text": "...", "label": 304},
+            ...
+        ]
+    }
+    """
+    texts_with_labels = json.loads(request.get_data(as_text=True))['data']
+    texts = map(lambda entry: entry['text'], texts_with_labels)
+    labels = map(lambda entry: entry['label'], texts_with_labels)
+
+    insert_into(DB_FILENAME, texts, labels, 'user')
+    n_inserts = len(texts_with_labels)
+
+    return jsonify({'n_inserted': n_inserts}), 201
+
+
 @app.route("/texts", methods=['GET'])
```

Comment:
da hier war ich mir noch nich sicher: sollten die n texts die mit der hoechsten uncertainty sein, entsprechend des letzten modells? was meinst du?

Reply:
Ja sollten sie denke ich auch. Macht daher wahrscheinlich Sinn, die Samples direkt vom Modell abfragen zu können.

Reply:
Oder das Modell trainiert sich regelmäßig neu und speichert Uncertainty values mit in die labels dB?

Reply:
Auch ne gute Idee